state.go

```diff
@@ -714,6 +714,35 @@ func (s *State) onInterface(se *Session, i interface{}) (err error) {
 		if s.TrackVoice {
 			err = s.voiceStateUpdate(t)
 		}
+	case *PresenceUpdate:
+		if s.TrackMembers {
+			var m *Member
+			m, err = s.Member(t.GuildID, t.User.ID)
```

Comment:
Even though you'd be shadowing the err, it should still be fine to do:

m, err := s.Member.....